bokehjs/src/lib/core/css.ts

```diff
@@ -394,7 +394,7 @@ type CSSStylesCamel = {
   /** @deprecated */
   wordWrap?: string | null
   writingMode?: string | null
-  zIndex?: string | null
+  zIndex?: string | number | null
```

Comment:
I like the strictness of always having the styles as strings; you don't need to guess if it can work as a number. 

Even if the type system will make it somewhat obvious, it is not everybody who has set it up to work in their IDE.

Reply:
To be clear, your preference is to revert this change? Which is fine, because I don't have much use for this.

Reply:
Yes, my preference is to be strict and only accept strings.